spring-commercetools/src/main/java/com/epam/esm/commercetools/client/CommerceToolsClient.java

```diff
@@ -0,0 +1,40 @@
+package com.epam.esm.commercetools.client;
+
+import com.commercetools.api.client.ProjectApiRoot;
+import com.commercetools.api.defaultconfig.ApiRootBuilder;
+import com.commercetools.api.defaultconfig.ServiceRegion;
+import io.vrap.rmf.base.client.oauth2.ClientCredentials;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
+
+@Configuration
+@PropertySource("classpath:application.yaml")
+public class CommerceToolsClient {
+
+    private final String clientId;
+    private final String clientSecret;
+    private final String projectKey;
+
+    @Autowired
+    public CommerceToolsClient(@Value("${commerce.clientId}") String clientId,
+                               @Value("${commerce.clientSecret}") String clientSecret,
```

Comment:
Commercetools client inside another Commercetools client?

Reply:
I don`t understand this one , it was done due to preparing constant for work

Reply:
Nvm, due to kotlin way of writing contructors, I didn't realize that this is a java constructor. You can ignore this one